Python/compile.c

```diff
@@ -1165,13 +1197,15 @@ stack_effect(int opcode, int oparg, int jump)
         /* Iterators and generators */
         case GET_AWAITABLE:
             return 0;
-        case SETUP_ASYNC_WITH:
+        case SETUP_WITH:
             /* 0 in the normal flow.
              * Restore the stack position to the position before the result
-             * of __aenter__ and push 6 values before jumping to the handler
+             * of __aenter__ and push 7 values before jumping to the handler
```

Comment:
```suggestion
             * of __aenter__ and push 4 values before jumping to the handler
```

Reply:
Also, is that supposed to be `__enter__` instead of `__aenter__`?

Reply:
Either, as it is preceded by either `BEFORE_WITH` or `BEFORE_ASYNC_WITH`.